tests/unit/ui/components/TestUserReview.js

```diff
@@ -49,7 +49,7 @@ describe(__filename, () => {
     });
     const root = render({ review, showRating: true });
 
-    expect(root.find('.UserReview-body').html()).toContain(fakeReview.body);
+    expect(root.find('.UserReview-body div').html()).toContain(fakeReview.body);
```

Comment:
The problem with asserting about what's rendered in the `div` is that we're now basing the test on the output of the `ShowMoreCard`, which could theoretically change. It's best not to test the output of components that we're using in other components. Ideally we'd just be checking the value of the `children` passed into the `ShowMoreCard`, because that's what this component does: it passes children into `ShowMoreCard` and then assumes that `ShowMoreCard` will do the correct thing.

The same goes for the other tests that you've updated.